### DISCUSSION TITLE
tests in sdist

### DISCUSSION THREAD

**Discussion by mtelka** [Kludex/uvicorn#2436] [Q&A]
I'm packaging uvicorn for OpenIndiana and I'd like to use sdist to run tests.  Would you be open to consider adding tests to the sdist?

Thank you.

**Comment by Kludex** [comment]
Yeah, just open a PR. 👍

**Reply by mtelka** [reply]
Great.  Any idea where are listed all test dependencies?  I failed to find any list in the git repo. :-(

Thank you.

**Reply by mtelka** [reply]
Obviously, there is the `requirements.txt` file.  But since it is missing `anyio` I just overlooked it :-).
